berkeley-function-call-leaderboard/model_handler/utils.py

```diff
@@ -10,6 +10,7 @@ def _cast_to_openai_type(properties, mapping, test_category):
     for key, value in properties.items():
         if "type" not in value:
             properties[key]["type"] = "string"
+            var_type = "string"
```

Comment:
These fixes are likely to change the leaderboard for LLMs that make use of the `mapping`, as it fixes a bug.

The bug is that we were trying to compare the type _after_ it was converted, so the compare always failed.